Shared/messages/MessageFeed.cpp

```diff
@@ -113,12 +113,39 @@ QFuture<DynamicEntityDataSourceInfo*> MessageFeed::onLoadAsync()
   auto* dynamicEntityDataSourceInfo = new DynamicEntityDataSourceInfo(entity_id_field_name, fields, this);
   dynamicEntityDataSourceInfo->setSpatialReference(SpatialReference::wgs84());
 
+  // create a function that can be passed on received updates to check for selection
+  auto checkEntityForSelectAction = [this](DynamicEntity* dynamicEntity)
```

Comment:
Can this be a member function? Otherwise it needs to be captured/copied and passed around.